model/project_aliases.go

```diff
@@ -252,14 +234,14 @@ func aliasesFromMap(input map[string]ProjectAliases) []ProjectAlias {
 // FindAliasInProjectRepoOrConfig finds all aliases with a given name for a project.
 // If the project has no aliases, the repo is checked for aliases.
 func FindAliasInProjectRepoOrConfig(projectID, alias string) ([]ProjectAlias, error) {
-	aliases, shouldExit, err := findAliasInProjectOrRepoFromDb(projectID, alias)
+	aliases, err := findAliasInProjectOrRepoFromDb(projectID, alias)
 	if err != nil {
 		return nil, errors.Wrap(err, "checking for existing aliases")
 	}
 	// If nothing is defined in the DB, check the project config,
 	// unless the alias defined is a patch alias and there are patch aliases
 	// defined on the project page.
-	if len(aliases) > 0 || shouldExit {
+	if len(aliases) > 0 {
```

Comment:
The 2nd part of the above comment should be removed.

Reply:
done